src/core/Entry.cpp

```diff
@@ -784,3 +789,20 @@ QString Entry::resolvePlaceholder(const QString& str) const
 
     return result;
 }
+
+QString Entry::resolveUrl(const QString& url) const
+{
+    QStringList parts = url.toLower().split("://");
```

Comment:
What happens if you ensure that the string ends with a slash if it doesn't contain any already?